### PR TITLE
Add `resume` command

### LINKED ISSUE #5: New command: `resume`
> Hello,
> 
> Would you consider adding a command to resume the renderer?
> 
> I know there's already a `pause` command that handles both pausing and resuming, but sometimes that is not enough, and it's also a bit confusing.
> 
> Here is the scenario I'm in (TL;DR below):
> 
> I'm running a small private server with a Raspberry Pi 4, using the Pl3xmap plugin for its webmap feature. However, the map has grown to 6GB, and if I keep Pl3xmap running continuously, it can crash the server while updating. To avoid this, I manually pause and resume it, which works well but is a hassle. I want to create a (scheduled bash) script that automatically resumes it at night and pauses it again after a set time. Using the current two-in-one command would be unreliable as it can sometimes accidentally resume it instead of pausing, or vice-versa. So, separate commands would be preferable in this scenario.
> 
> TL;DR: The two-in-one command we currently have is unpredictable and unintuitive, and cannot be reliably used in automation. Separate commands would solve those issues.

@hmlendea

### PR DESCRIPTION
**Pull request by jedrek0429**
Resolves #5.

### CODE REVIEW

File: core/src/main/java/net/pl3x/map/core/command/CommandHandler.java [hunk 1]
```diff
@@ -112,6 +113,7 @@ default void registerSubcommands() {
                 new ShowCommand(this),
                 new StatusCommand(this),
                 new StitchCommand(this),
+                new UnpauseCommand(this),
```
Comment (granny) [inline core/src/main/java/net/pl3x/map/core/command/CommandHandler.java]: nitpick: move underneath the `PauseCommand` call

**Review by granny — CHANGES_REQUESTED** [review]
replace the `unpause` phrase everywhere with `resume` instead

File: core/src/main/java/net/pl3x/map/core/command/CommandHandler.java (hunk 1 above)
Comment (jedrek0429) [reply core/src/main/java/net/pl3x/map/core/command/CommandHandler.java:116]: I just thought they were sorted alphabetically, but I'll change that.

Comment (jedrek0429) [reply core/src/main/java/net/pl3x/map/core/command/CommandHandler.java]: b8bb439

File: core/src/main/java/net/pl3x/map/core/configuration/Lang.java [hunk 2]
```diff
@@ -141,6 +148,7 @@ public final class Lang extends AbstractConfig {
     @Key("command.stitch.finished")
     public static String COMMAND_STITCH_FINISHED = "<green>Finished stitching <count> tiles!\n<green>You can find it at <grey>/tiles/<world>/stitched/<filename>";
 
+    
     @Key("command.version.description")
```
Comment (granny) [inline core/src/main/java/net/pl3x/map/core/configuration/Lang.java]:
```suggestion

    @Key("command.version.description")
```

File: core/src/main/java/net/pl3x/map/core/configuration/Lang.java [hunk 3]
```diff
@@ -93,11 +93,18 @@ public final class Lang extends AbstractConfig {
...
-    public static String COMMAND_PAUSE_UNPAUSED = "<green>Renderers are now unpaused";
+    @Key("command.pause.already-paused")
+    public static String COMMAND_PAUSE_ALREADY_PAUSED = "<grey>Renderers are already paused";
+
+    @Key("command.resume.description")
+    public static String COMMAND_RESUME_DESCRIPTION = "Resume renderers";
+    @Key("command.resume.resumed")
+    public static String COMMAND_RESUME_RESUMED = "<green>Renderers are now resumed";
+    @Key("command.resume.already-resumed")
+    public static String COMMAND_RESUME_ALREADY_RESUMED = "<grey>Renderers are already resumed";
```
Comment (granny) [inline core/src/main/java/net/pl3x/map/core/configuration/Lang.java]:
```suggestion
    @Key("command.resume.resumed")
    public static String COMMAND_RESUME_RESUMED = "<green>Renderers have now resumed";
    @Key("command.resume.already-resumed")
    public static String COMMAND_RESUME_ALREADY_RESUMED = "<grey>Renderers have already resumed";
```

**Review by granny — APPROVED** [review]
nitpick
